### PR TITLE
Added frequently used data structures in c++ 

### PR DESCRIPTION
**Pull request by adityaajay29**
Previously, only vector, stack and queue data structures were present in the c++ document. I have added priority queue, set, unordered set, multiset, map and unordered map to the list, and have also formatted the document a bit. All the codes are properly formatted with a good explanation of all data structures.

### CODE REVIEW

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]
These belong to DS instead the language cheat sheet.

**Comment by onecompiler-ops** [comment]
We think this is a little verbose for c++ cheat sheet. 
This is a very specific use-case, and does not belong in Cheat Sheets, please write this as a post instead here https://onecompiler.com/posts

**Comment by adityaajay29** [comment]
Thanks for the review, I will remove the descriptive paragraphs of these DS, and make a short table instead for a quick glimpse of c++ STL. Is that fine?

**Comment by adityaajay29** [comment]
Done! Please review it.

File: cpp.md [hunk 1]
```diff
@@ -311,75 +319,7 @@ int main()
 }
 
 ```
-## Stacks
```
Comment (onecompiler-ops) [inline cpp.md:314]: Why this got deleted?

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

File: cpp.md (hunk 1 above)
Comment (adityaajay29) [reply cpp.md:314]: As stack and queue are one of the DS only, so only including these two seemed incomplete and inconsistent. If you say, I'll undo that change.

**Pull request closed without merging**